### PR TITLE
benchmarks: fix create test

### PR DESCRIPTION
**Pull request by vrothberg**
And a new one for `run --detach`.

Signed-off-by: Valentin Rothberg <vrothberg@redhat.com>

#### Does this PR introduce a user-facing change?

```release-note
None
```
@mheon @Luap99 @rhatdan PTAL

We can add more benchmarks in the future.

### CODE REVIEW

File: test/e2e/benchmarks_test.go [hunk 1]
```diff
@@ -262,5 +262,11 @@ var _ = Describe("Podman Benchmark Suite", func() {
 			session.WaitWithDefaultTimeout()
 			Expect(session).Should(Exit(0))
 		}, nil)
+
+		newBenchmark("podman run --dete", func() {
```
Comment (Luap99) [inline test/e2e/benchmarks_test.go:266]:
```suggestion
		newBenchmark("podman run --detach", func() {
```

Comment (vrothberg) [inline test/e2e/benchmarks_test.go:266]:
```suggestion
		newBenchmark("podman run --dete", func() {
```
Typo - will fix.

**Comment by vrothberg** [comment]
### Podman v4.1.1
```
Ran 3 samples:                  
[CPU] podman create:            
  Fastest Time: 0.134s          
  Slowest Time: 0.144s          
  Average Time: 0.141s ± 0.005s 
[MEM] podman create:            
  Smallest: 43072.0KB           
   Largest: 43940.0KB           
   Average: 43423.3KB ± 276.7KB 
[CPU] podman start:             
  Fastest Time: 0.134s          
  Slowest Time: 0.144s          
  Average Time: 0.139s ± 0.005s 
[MEM] podman start:             
  Smallest: 42376.0KB           
   Largest: 43112.0KB           
   Average: 42716.7KB ± 251.3KB 
[CPU] podman run:               
  Fastest Time: 0.133s          
  Slowest Time: 0.235s          
  Average Time: 0.155s ± 0.036s 
[MEM] podman run:               
  Smallest: 44548.0KB           
   Largest: 45296.0KB           
   Average: 44992.7KB ± 252.5KB 
[CPU] podman run --dete:        
  Fastest Time: 0.133s          
  Slowest Time: 0.153s          
  Average Time: 0.138s ± 0.007s 
[MEM] podman run --dete:        
  Smallest: 44204.0KB           
   Largest: 45624.0KB           
   Average: 44776.7KB ± 504.3KB 
```

### HEAD
```
Ran 3 samples:                    
[CPU] podman create:              
  Fastest Time: 0.133s            
  Slowest Time: 0.144s            
  Average Time: 0.136s ± 0.004s   
[MEM] podman create:              
  Smallest: 41432.0KB
   Largest: 42848.0KB
   Average: 42448.7KB ± 495.9KB
[CPU] podman start:
  Fastest Time: 0.134s
  Slowest Time: 0.134s
  Average Time: 0.134s ± 0.000s
[MEM] podman start:
  Smallest: 42180.0KB
   Largest: 43236.0KB
   Average: 42886.0KB ± 396.0KB
[CPU] podman run:
  Fastest Time: 0.337s
  Slowest Time: 0.338s
  Average Time: 0.337s ± 0.000s
[MEM] podman run:
  Smallest: 44932.0KB
   Largest: 45532.0KB
   Average: 45194.7KB ± 184.8KB
[CPU] podman run --dete:
  Fastest Time: 0.132s
  Slowest Time: 0.134s
  Average Time: 0.134s ± 0.001s
[MEM] podman run --dete:
  Smallest: 43740.0KB
   Largest: 45748.0KB
   Average: 44681.3KB ± 673.8KB
```

**Review by Luap99 — COMMENTED** [review]
LGTM

**Comment by mheon** [comment]
This LGTM. I don't think we harmed actual user responsiveness, so `podman run` taking slightly longer to exit shouldn't matter.

**Pull request merged**